SimplenoteUITests/EmailLogin.swift

```diff
@@ -14,7 +14,18 @@ class EmailLogin {
         backButton.tap()
     }
 
-    class func logIn(email: String, password: String) {
+    class func logIn() {
+        var testAccount = ProcessInfo.processInfo.environment["UI_TEST_ACCOUNT"]!
```

Comment:
It seems like using force unwrap is not a problem here, because the environment variable exists in the scheme for sure.

Reply:
For some reason, opening the project in Xcode removed the scheme change and the test crashed here.

Even without the Xcode quirk I experienced, I find it's best not to make assumptions on input parameters in the code. This one in particular lives in a scheme in the form of a string. There is no compile-time check that would tell a developer that removed it that it was expected here.

I would suggest explicitly handling the nil scenario. Something like

```swift
let testAccountKey = "UI_TEST_ACCOUNT"
let testAccount: String
switch ProcessInfo.processInfo.environment[testAccountKey] {
case .none:
    fatalError("Expected \(testAccountKey) environment variable to be defined in the scheme")
case .some(let value):
    // Use 'default' account if test account was not passed via environment variable
    testAccount = value.isEmpty ? testDataEmail : value
}
```

This way, it something goes wrong, like it happened to me, the error message in the console will be more informative.
<img width="1392" alt="Screen Shot 2021-05-05 at 1 06 35 pm" src="https://user-images.githubusercontent.com/1218433/117093399-2f048800-ada4-11eb-9653-a6c5e04ebe06.png">

Reply:
Thank you! I agree, another reason of doing it the way you suggest is that tests might be executed using the other scheme. I used you code 🙇

Reply:
Thank you! I agree, another reason of doing it the way you suggest is that tests might be executed using the other scheme. I used you code 🙇